Lib/pdb.py

```diff
@@ -1368,6 +1371,9 @@ def do_source(self, arg):
             return
         try:
             lines, lineno = inspect.getsourcelines(obj)
+            # inspect.getsourcelines() returns lineno = 0 for
+            # module-level frame which breaks our code print line number
+            lineno = max(1, lineno)
```

Comment:
Make sure we have an issue for the inspect bug, and put the number of that issue in this comment.

Reply:
Okay got it. I believe @ortem was working on a separate issue for `inspect`. I don't want to step on his toes so I'll wait for that issue.

Reply:
Thanks for the reminder, I have just opened a separate issue #103319 so you can put it in the comment